src/test/java/bio/terra/integration/DrsTest.java

```diff
@@ -0,0 +1,117 @@
+package bio.terra.integration;
+
+import bio.terra.category.Integration;
+import bio.terra.integration.auth.AuthService;
+import bio.terra.integration.configuration.TestConfiguration;
+import bio.terra.model.DRSAccessMethod;
+import bio.terra.model.DRSChecksum;
+import bio.terra.model.DRSObject;
+import bio.terra.model.FSObjectModel;
+import bio.terra.model.SnapshotModel;
+import bio.terra.model.SnapshotSummaryModel;
+import com.google.cloud.bigquery.BigQuery;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest // TODO: Do we need this?
```

Comment:
I don't think we are using any of the [features this enables](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/test/context/SpringBootTest.html).